src/api/Nhs.Appointments.Api/Consumers/BookingMadeConsumer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MassTransit;
+using Nhs.Appointments.Api.Notifications;
+using Nhs.Appointments.Core.Messaging.Events;
+using System;
+using System.Threading.Tasks;
+
+namespace Nhs.Appointments.Api.Consumers;
+
+public class BookingMadeConsumer(IBookingMadeNotifier notifier) : IConsumer<BookingMade>
+{
+    public Task Consume(ConsumeContext<BookingMade> context)
+    {
+        return notifier.Notify(
+            nameof(BookingMade),
+            context.Message.Service,
+            context.Message.Reference,
+            context.Message.Site,
+            context.Message.FirstName,
+            DateOnly.FromDateTime(context.Message.From),
+            TimeOnly.FromDateTime(context.Message.From),
+            context.Message.EmailContactConsent,
```

Comment:
This pattern looks like it wouldn't extend to other notification types. Could we have an array of contacts made up of a contact type and contact details. Presence of the contact information would therefore imply consent. So if the user had supplied email and phone the list would contain

[
{"type": "email", target:"vicr1@hscic.gov.uk"}
{"type": "phone", target:"07876345876"}
]
if they supplied only one then only the one they wanted would appear. This would allow us to support other notification types more readily in the future if they were required (e.g. NHS App, WhatsApp, Discord, Slack) It would also allow us to support sending to multiple email addresses or phone numbers if that became a thing